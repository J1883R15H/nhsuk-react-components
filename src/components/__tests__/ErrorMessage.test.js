import React from 'react';
import { shallow } from 'enzyme';
import ErrorMessage from '../error-message';

describe('ErrorMessage', () => {
  it('matches snapshot', () => {
    const shallowMount = shallow(<ErrorMessage>Error Message</ErrorMessage>);
    expect(shallowMount).toMatchSnapshot();
    shallowMount.unmount();
  });
});
