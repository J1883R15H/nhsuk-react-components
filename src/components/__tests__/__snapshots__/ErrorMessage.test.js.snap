// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorMessage matches snapshot 1`] = `
<span
  className="nhsuk-error-message "
  style={Object {}}
>
  <span
    className="nhsuk-u-visually-hidden"
  >
    Error: 
  </span>
  Error Message
</span>
`;
